src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/Baselines/ServiceCollection/Configure_T.net462.generated.txt

```diff
@@ -0,0 +1,243 @@
+// <auto-generated/>
```

Comment:
Thanks for adding the `net462` variants. Could we refactor to the following structure?

```
> baselines
    > netcoreapp
        > foo.generated.txt
    > net462
        > foo.generated.txt
```

I'd actually suggest making a preliminary PR doing this. We'd need to remove the condition on [[this line]](https://github.com/dotnet/runtime/blob/e80c9aedce4e8d9bdead812cfeedb6c9a2428aec/src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/Microsoft.Extensions.Configuration.Binder.SourceGeneration.Tests.csproj#L46) so that the net462 tests would be included in automated runs. Doing this will reveal a bit of additional, tangential work needed to complete the effort.